sos/plugins/grafana.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2016 Red Hat, Inc., Pratik Bandarkar <pbandark@redhat.com>
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+from sos.plugins import Plugin, RedHatPlugin
+
+
+class Grafana(Plugin, RedHatPlugin):
+    """Grafana
+    """
+    plugin_name = "grafana"
+    profiles = ('services',)
+
+    def setup(self):
+        self.limit = self.get_option("log_size")
+        if self.get_option("all_logs"):
+            self.add_copy_spec_limit("/var/log/grafana/",
+                                     sizelimit=self.limit)
```

Comment:
If you're doing `all_logs` there should be no size limits applied.. (it literally means "all logs").